pymatgen/io/pwmat/outputs.py

```diff
@@ -190,12 +190,12 @@ def __init__(self, filename: PathLike):
         self._eigenvalues = self._parse_eigen()
         self._kpts, self._kpts_weight, self._hsps = self._parse_kpt()
 
-    def _parse_band(self) -> tuple[int, int, int]:
+    def _parse_band(self) -> Tuple3Ints:
         """Parse REPORT file to obtain spin switches, the number of kpoints
         and the number of bands.
 
         Returns:
-            tuple[int, int, int]: containing:
+            Tuple3Ints: containing:
```

Comment:
i think for doc strings it's better to keep the explicit type since they're rendered as [HTML in the API docs](https://pymatgen.org/pymatgen.html) where people would need to go hunting for the definition of that type variable to understand what it means

Reply:
Done in 64924f3e8f3b66d5b5a9c9d0184e1b6b38a06e7f. Thanks for the comment :)